ingest/src/test/java/com/microsoft/azure/kusto/ingest/E2ETest.java

```diff
@@ -151,8 +158,12 @@ private static boolean isManualExecution() {
 
     private static void createTableAndMapping() {
         try {
-            queryClient.executeToJsonResult(DB_NAME, String.format(".drop table %s ifexists", tableName));
+            queryClient.executeToJsonResult(DB_NAME, String.format(".drop table %s ifexists ", tableName));
```

Comment:
NIT: why the trailing space?

Reply:
fixed. sometimes I copy paste things that I should really just type and then extra spaces get in there.